analytics_data_api/v0/views/learners.py

```diff
@@ -223,7 +233,13 @@ def get_queryset(self):
         # parameter values in `get_users_in_course`.
         params = {key: val for key, val in params.items() if val is not None}
         try:
-            return RosterEntry.get_users_in_course(self.course_id, **params)
+            users = RosterEntry.get_users_in_course(self.course_id, **params)
+            last_updated = {'date': None}
+            if len(RosterUpdate.get_last_updated()) == 1:
+                last_updated = RosterUpdate.get_last_updated()[0]
+            else:
+                logger.warn('RosterUpdate not found.')
```

Comment:
The intension is for for update index to complement the learn index, and my thinking here is that it doesn't make sense for the entire call to fail if the update index isn't available for some reason.  What do you think?